src/modules/status.rs

```diff
@@ -684,4 +689,26 @@ mod tests {
             assert_eq!(expected, actual);
         }
     }
+
+    #[test]
+    fn pipestatus_segment_format() {
+        let pipe_exit_code = &[0, 1];
+        let main_exit_code = 1;
+
+        let expected = Some("[0]|[1] => [1]".to_string());
+        let actual = ModuleRenderer::new("status")
+            .config(toml::toml! {
+                [status]
+                format = "\\($status\\)"
+                pipestatus = true
+                pipestatus_separator = "|"
+                pipestatus_format = "$pipestatus => \\[$status\\]"
```

Comment:
Please use a different style for the final `$status` in `pipestatus_format`
```suggestion
        let expected = Some("[0]|[1] => <1>".to_string());
        let actual = ModuleRenderer::new("status")
            .config(toml::toml! {
                [status]
                format = "\\($status\\)"
                pipestatus = true
                pipestatus_separator = "|"
                pipestatus_format = "$pipestatus => <$status>"
```